doc/introduction/index.rst

```diff
@@ -408,6 +411,28 @@ The Boot process proceeds as follows:
    services shall not use IOMMU. In addition, currently we only support
    UEFI timer with HPET MSI.
 
+In this boot mode, both the Service and User VM boot options (e.g. Linux
+command-line parameters) are configured following the instructions for the EFI
+bootloader used by the Operating System (OS).
+
+* In the case of Clear Linux, this EFI bootloader is `systemd-boot`_ and Linux
+  kernel command-line parameters are defined in the ``.conf`` files.
+* Another popular EFI bootloader used by Linux distributions is `grub`_.
+  Distributions like Ubuntu/Debian, Fedora/CentOS use `grub`_
+
+.. note::
+
+   The `Slim Bootloader <https://www.intel.com/content/www/us/en/design/products-and-solutions/technologies/slim-bootloader/overview.html>`_
+   is an alternative boot firmware that can be used to boot ACRN. The `Boot ACRN Hyervisor <https://slimbootloader.github.io/how-tos/boot-acrn.html>`_
+   and :ref:`using-sbl-up2` tutorials have more information on how to use SBL with ACRN.
+
+.. note::
+
+   A virtual `Slim Bootloader <https://www.intel.com/content/www/us/en/design/products-and-solutions/technologies/slim-bootloader/overview.html>`_
+   can also be used to start User VMs, it is called ``vSBL``. The :ref:`acrn-dm_parameters` has more information
+   on how to boot a User VM using ``vSBL``. Note that in this case, the kernel command-line parameters are
+   defined directly in the launch script, via the ``-B`` option.
```

Comment:
@gvancuts 
sorry for my mistake, if the bootloader is vSBL, the cmdline is also defined in .conf file, i.e. it is same as OVMF. So just remove the phrase "Note that in this case, the kernel command-line parameters are defined directly in the launch script, via the ``-B`` option.".

The "-B" option is needed only when "-k" parameter specified.

Reply:
Thanks for the review @jsun26intel . I will remove this reference to the `-B` option (or perhaps I should really also add it as a third method one can use to boot a User VM, or is being deprecated?). If I understand it correctly, there are three different methods for launching a post-launched User VM:
1. Using `OVMF.fd` (virtual EFI firmware - cmdline defined in the User VM OS bootloader)
2. Using `vSBL` (cmdline defined in the `cmdline.txt` file passed on to the `iasimage` script)
3. Using a kernel directly (with the `-k` option, in which case the cmdline is defined using the `-B` option)
Can you confirm that this is correct and if so, that the third option is here to stay?

I think your statement is slightly incorrect in that if one uses the `vSBL` firmware, the kernel command-line parameters are defined in the `cmdline.txt` file passed on to the `iasimage` script (and not a conf file). Can you confirm?

Reply:
@binbinwu1 
please confirm UOS cmdline on vSBL path.

Reply:
@jsun26intel Yes, for vSBL, launch script should add -B option to pass paramters to UOS.

Reply:
@gvancuts 
Confirmed with binbin,  the cmdline should be combination of cmdline.txt and -B option for vSBL path, and actually the cmdline.txt is null with current implementation.
Synced with Michael, the vSBL support is deprecated, do we need to mention them in doc?